modules/openapi-generator/src/main/resources/cpp-qt5-client/HttpRequest.cpp.mustache

```diff
@@ -347,12 +351,14 @@ void {{prefix}}HttpRequestWorker::execute({{prefix}}HttpRequestInput *input) {
         buffer->setParent(reply);
 #endif
     }
-    if (_timeOut > 0) {
-        QTimer::singleShot(_timeOut, [=]() { on_manager_timeout(reply); });
+    if (timeOutTimer.interval() > 0) {
+        timeOutTimer.callOnTimeout([=]() { on_manager_timeout(reply); });
```

Comment:
This is available only on 5.12 and above. Is there a possibility to use a more backward compatible call?